docs/usage/shoot-operations/worker_pool_manual_rotation.md

```diff
@@ -0,0 +1,17 @@
+# Manual Worker Pool Rollout
+
+There may be cases when an end user might want to trigger a manual worker pool rollout. For example, the dual-stack migration requires to roll nodes.
+This can be accomplished by annotating the shoot with the `rollout-workers` annotation and specifying which worker pools you'd like to be rolled out.
+
+```bash
+kubectl -n <shoot-namespace> annotate shoot <shoot-name> gardener.cloud/operation=rollout-workers=<pool1-name>[,<pool2-name>,...]
+```
+
+Alternatively, you can use `*` to roll out all worker pools:
+
+```bash
+kubectl -n <shoot-namespace> annotate shoot <shoot-name> gardener.cloud/operation=rollout-workers=*
+```
```

Comment:
Have you tried running this command? The shell might very well interpret `*` so that this command should be quoted.

Reply:
I tried it in `zsh` without any folder/file matching to the expression:

```
% echo gardener.cloud/operation=rollout-workers=*
zsh: no matches found: gardener.cloud/operation=rollout-workers=*
```

In `bash` it indeed worked, but I agree it is probably safer to add quotes to the documentation, e.g. the following worked in `zsh` and `bash`:

```
% echo "gardener.cloud/operation=rollout-workers=*"
gardener.cloud/operation=rollout-workers=*
```